packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -251,19 +323,63 @@ const Select = <
             return null;
           }
 
-          return <Chip key={item} label={selectedOption.text} />;
+          return (
+            <MuiChip
+              key={item}
+              label={
+                <>
+                  {selectedOption.text}
+                  {!isPlaceholder &&
+                    controlledStateRef.current === CONTROLLED &&
+                    hasMultipleChoices && (
+                      <PlaceholderIcon
+                        odysseyDesignTokens={odysseyDesignTokens}
+                      />
```

Comment:
What's `isPlaceholder` doing here? I would expect it to mean it's placeholder text. But that doesn't seem to follow the logic of adding a placeholder icon when `isPlaceholder` is `false`.